src/Epinova.ElasticSearch.Core.EPiServer/Controllers/Abstractions/ElasticSearchControllerBase.cs

```diff
@@ -111,7 +120,7 @@ protected override void Initialize(RequestContext requestContext)
 
         private List<IndexInformation> GetIndices()
         {
-            var indexHelper = new Admin.Index(_serverInfoService, _settings, _httpClientHelper, "GetIndices-NA");
+            var indexHelper = new Admin.Index(_serverInfoService, _settings, _httpClientHelper, null);
```

Comment:
Når man sitter og leser kode som her i nettleser, er det til stor nytte å prefikse med navn på argument for null, tall, bool osv: Altså fortell meg hva som er navnet på argumentet som får verdien null.